project.clj

```diff
@@ -4,12 +4,25 @@
   :license {:name "LGPLv3"
             :url  "https://github.com/carocad/parcera/blob/master/LICENSE.md"}
   :dependencies [[org.clojure/clojure "1.10.1"]
+                 [org.clojure/test.check "0.10.0"]
+                 [criterium/criterium "0.4.5"] ;; benchmark
```

Comment:
please dont do this. This would make the project upstream import these dependencies when using `parcera` even though both of these deps are only used for testing

Reply:
We'll have to figure out how to make `lein install` work without it then.

Reply:
I figured it out.

Reply:
Nice :) !